console/console-init/ui/src/modules/project/utils/utils.ts

```diff
@@ -269,51 +269,57 @@ const getQueryVariableForCreateMessagingProject = (
   if (customizeEndpoint) {
     const endpoints: IExposeEndPoint[] = [];
     if (protocols && protocols.length > 0) {
-      protocols.map((protocol: string) => {
-        const endpoint: IExposeEndPoint = { service: "messaging" };
-        if (protocol === EndPointProtocol.AMQPS) {
-          endpoint.name = "messaging";
-        } else if (protocol === EndPointProtocol.AMQP_WSS) {
-          endpoint.name = "messaging-wss";
-        }
-        if (tlsCertificate) {
-          endpoint.certificate = {
-            provider: tlsCertificate
-          };
-          if (
-            tlsCertificate === TlsCertificateType.UPLOAD_CERT &&
-            certValue &&
-            certValue.trim() !== "" &&
-            privateKey &&
-            privateKey.trim() !== ""
-          ) {
+      try {
+        protocols.map((protocol: string) => {
```

Comment:
map should be used only when you want to return an array. For cases when you're not returning anything, use forEach.